newIDE/app/src/UI/ResponsiveImagesGallery.js

```diff
@@ -206,18 +213,17 @@ const ResponsiveImagesGallery = ({
           String(selectedImageIndex + 1)
         )}
       />
-      <Grid
-        classes={classesForGridContainer}
-        container
-        spacing={GRID_SPACING}
-        wrap="nowrap"
-        style={styles.grid}
-      >
+      <div style={styles.desktopGallery}>
         {imagesUrls.map((url, index) => (
-          <Grid
-            item
+          <div
```

Comment:
Good question, the gallery has another rendering for mobile/small screens which I left untouched, earlier in the code (pretty sure it could be simplified, but no time to change because it works well)